internal/driver/gomobile/repository.go

```diff
@@ -69,5 +73,20 @@ func (m *mobileFileRepo) Reader(u fyne.URI) (fyne.URIReadCloser, error) {
 }
 
 func (m *mobileFileRepo) Writer(u fyne.URI) (fyne.URIWriteCloser, error) {
+	if u == nil || u.Scheme() != "file" {
+		return nil, repository.ErrOperationNotSupported
```

Comment:
We can write to URIs that are not file, so this check isn't right.
The workaround we need is for when a file write fails with file not found (or equivalent)